threenums.c

```diff
@@ -0,0 +1,22 @@
+#include <stdio.h>
```

Comment:
Doesn't handle all input correctly! For example, when you input 10 100 10, it outputs 10 as the largest number. I'm pretty sure this is because you didn't consider the cases in which 2 numbers are equal. With that said, this is why you should include comments (so I can understand your code and easily figure out where you went wrong)